classes/class-wc-connect-api-client.php

```diff
@@ -135,6 +135,15 @@ public function get_shipping_rates( $services, $package ) {
 
 
 		/**
+		 * Asks the WooCommerce connect server for an array of payment methods
+		 *
+		 * @return true|WP_Error
```

Comment:
Fixed in https://github.com/Automattic/woocommerce-connect-client/pull/488/commits/8ebe257dc446a63b65a6a909757e8a51ee1eb35a